### PR TITLE
Improve translatability of memstick screen

### PR DESCRIPTION
**Pull request by hrydgard**
@iota97 I think this solves the issue from 83b6139

### CODE REVIEW

File: UI/MemStickScreen.cpp [hunk 1]
```diff
@@ -134,28 +134,33 @@ void MemStickScreen::CreateViews() {
...
+	leftColumn->Add(new TextView(iz->T("DataCanBeShared", "Data can be shared between PPSSPP regular/Gold.")))->SetBullet(true);
+	leftColumn->Add(new TextView(iz->T("EasyUSBAccess", "Easy USB access")))->SetBullet(true);
 
 	leftColumn->Add(new Choice(iz->T("Use App Private Directory")))->OnClick.Handle(this, &MemStickScreen::OnUseInternalStorage);
-	leftColumn->Add(new TextView(iz->T("InternalStorageDesc", "* Warning! Data will be deleted if you uninstall PPSSPP!\n* Data cannot be shared between PPSSPP regular/Gold\n* USB access through "
-#ifdef GOLD
-	"Android/data/org.ppsspp.ppssppgold/files"
+	// Consider https://www.compart.com/en/unicode/U+26A0 (unicode warning sign?)? or a graphic?
+	leftColumn->Add(new TextView(iz->T("DataWillBeLostOnUninstall", "Warning! Data will be lost when you uninstall PPSSPP!")))->SetBullet(true);
+	leftColumn->Add(new TextView(iz->T("DataCannotBeShared", "Data cannot be shared between PPSSPP regular / Gold!")))->SetBullet(true);
```
Comment (iota97) [inline UI/MemStickScreen.cpp:158]: nit: `regular / Gold` have space before and after the slash (in the other string is `regular/Gold`). 

Also you may want `cannot` in all caps for emphasis

File: UI/MemStickScreen.cpp [hunk 2]
```diff
@@ -134,28 +134,33 @@ void MemStickScreen::CreateViews() {
...
+#if GOLD
+	leftColumn->Add(new TextView(iz->T("USBAccessThrough", "USB access through Android/data/org.ppsspp.ppssppgold/files")))->SetBullet(true);
```
Comment (iota97) [inline UI/MemStickScreen.cpp:160]: Mhhh I don't think this fix the problem `USBAccessThrough` is the key of both, this mean when a translator add a translation he have no way to set the 2 different string in the ini (unless I'm missing something about how the ini work?). Better use `GoldUSBAccessThrough` for the gold version.

Comment (hrydgard) [reply UI/MemStickScreen.cpp]: Fixed

File: UI/MemStickScreen.cpp (hunk 1 above)
Comment (hrydgard) [reply UI/MemStickScreen.cpp]: Fixed

**Pull request merged**